src/dotnet/APIView/APIView/Languages/CodeFileBuilder.cs

```diff
@@ -68,7 +68,7 @@ private IEnumerable<INamespaceSymbol> EnumerateNamespaces(IAssemblySymbol assemb
             }
         }
 
-        public CodeFile Build(IAssemblySymbol assemblySymbol, bool runAnalysis)
+        public CodeFile Build(IAssemblySymbol assemblySymbol, bool runAnalysis, string[] dependencies)
```

Comment:
It's a bit strange to pass this value in.

Reply:
Why? Do you have an alternative?

Reply:
CodeFileBuilder is currently, only processing assemblies. Because of how the code is factored you have to pass additional package-level information in to be able to include it in the CodeFile.

One option would be to move the entire logic of nupkg parsing here.

Reply:
I think this would impact some of the legacy code paths where we pass in the assembly via args

Reply:
I see your point, but I also don't think passing in dependencies here is crazy